### PR TITLE
Remove ProblemSpec.pluginLocation()

### PR DESCRIPTION
**Pull request by donat**
Fixes [#30564](https://github.com/gradle/gradle/issues/30564)

### CODE REVIEW

**Comment by donat** [comment]
@bot-gradle test this

**Review by reinsch82 — APPROVED** [review]
LGTM

**Comment by donat** [comment]
@bot-gradle merge

File: platforms/ide/problems-api/src/main/java/org/gradle/api/problems/internal/DefaultPluginIdLocation.java [hunk 1]
```diff
@@ -23,7 +23,7 @@
  */
 public class DefaultPluginIdLocation implements PluginIdLocation {
 
-    private final String pluginId;
+    private final String pluginId;
```
Comment (MGaetan89) [inline platforms/ide/problems-api/src/main/java/org/gradle/api/problems/internal/DefaultPluginIdLocation.java:26]:
```suggestion
    private final String pluginId;
```

**Pull request merged**